modules/config-fields-misc.adoc

```diff
@@ -40,7 +40,9 @@
  +
 **Default:** 10
 | **V1_PUSH_WHITELIST** | Array of String | The array of namespace names that support V1 push if FEATURE_RESTRICTED_V1_PUSH is set to true
-| **V2_PAGINATION_SIZE**  | Number | The number of results returned per page in V2 registry APIs
+| **V2_PAGINATION_SIZE**  | Number | The number of results returned per page in V2 registry APIs +
+ +
+ **Default:** 100
```

Comment:
the default value is 50, see https://github.com/quay/quay/blob/master/config.py(Line 642)